astro/src/content/docs/lifecycle/authenticate-users/passwordless/webauthn.mdx

```diff
@@ -6,15 +6,15 @@ section: lifecycle
 subcategory: authenticate users
 tertcategory: passwordless
 ---
-import AdvancedEditionBlurb from 'src/content/docs/_shared/_advanced-edition-blurb.astro';
+import LicensedEditionBlurb from 'src/content/docs/_shared/_licensed-edition-blurb.astro';
 import Aside from 'src/components/Aside.astro';
 import AdvancedEditionBlurbApi from 'src/content/docs/_shared/_advanced-edition-blurb-api.astro';
```

Comment:
There is another callout that is still using this that seems it should be switched to `LicensedEditionBlurbApi`